library/pk_ecc.c

```diff
@@ -0,0 +1,304 @@
+#include "common.h"
```

Comment:
We need the usual copyright+licence lines. Probably should move the description here too ("ECC setters" below).